css-transitions/styles.css

```diff
@@ -0,0 +1,54 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245 245 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: 500ms;
```

Comment:
Leaving out the transition property defaults to transitioning all properties, which is generally not efficient, so you should specify what to transition

```css
transition: background-color 500ms;
```